src/mon/PGMap.cc

```diff
@@ -2589,7 +2589,7 @@ void PGMap::get_health_checks(
     switch(i.first) {
       case UNAVAILABLE:
         health_code = "PG_AVAILABILITY";
-        sev = HEALTH_WARN;
+        sev = HEALTH_ERR;
```

Comment:
In general this should be fine but I am not sure peering PGs should cause a HEALTH_ERR , what do you think?

Reply:
https://github.com/ceph/ceph/blob/master/src/mon/PGMap.cc#L2456

^^ based on that we only mark peering as UNAVAILABLE if it is stuck in that state for longer than mon_pg_stuck_threshold
Shouldn't that be acceptable?

Otherwise, I agree of course that temporary peering states shouldn't trigger HEALTH_ERR, (or WARN, for that matter).

Reply:
Experimentally, I find that we get HEALTH_WARN from `peering` e.g. if I mark an osd out. So something might be broken in the stuck peering logic I referred to.

```
2021-07-09 11:29:17.591946 mon.cephdwight-mon-1633994557 [INF] Client client.admin marked osd.1 out, while it was still marked up
2021-07-09 11:29:19.692510 mon.cephdwight-mon-1633994557 [WRN] Health check failed: Reduced data availability: 12 pgs peering (PG_AVAILABILITY)
2021-07-09 11:29:21.720761 mon.cephdwight-mon-1633994557 [WRN] Health check failed: Degraded data redundancy: 29/112379112 objects degraded (0.000%), 1 pg degraded (PG_DEGRADED)
2021-07-09 11:29:25.744896 mon.cephdwight-mon-1633994557 [INF] Health check cleared: PG_AVAILABILITY (was: Reduced data availability: 19 pgs peering)
2021-07-09 11:29:25.744929 mon.cephdwight-mon-1633994557 [INF] Health check cleared: PG_DEGRADED (was: Degraded data redundancy: 29/112379133 objects degraded (0.000%), 1 pg degraded)
2021-07-09 11:29:25.744939 mon.cephdwight-mon-1633994557 [INF] Cluster is now healthy
```

Reply:
@neha-ojha I submitted a tracker about the stuck peering issue https://tracker.ceph.com/issues/51688
We can put this on hold until that is fixed

Reply:
yeah, that makes sense